src/sources/forus-platform/pug/tpl/directives/website/navbar.pug

```diff
@@ -0,0 +1,103 @@
+nav.navbar.navbar-inverse
+  .container-fluid.cf-wr
+    .navbar-header
+      button.navbar-toggle(type='button' data-toggle='collapse' data-target='#myNavbar')
+        span.icon-bar
+        span.icon-bar
+        span.icon-bar
+      a.navbar-brand(href="/#!/")
+            include ../../../../resources/platform-general/assets/img/svg/forus-logo.svg
+    #myNavbar.collapse.navbar-collapse
+      ul.nav.navbar-nav
+        li.dropdown
+          a.dropdown-toggle(ui-sref="platform")
+            | Platform Forus
+        //-span.caret
+          ul.dropdown-menu
+            li
+              a(href='#') Page 1-1
+            li
+              a(href='#') Page 1-2
+            li
+              a(href='#') Page 1-3
+        li
+          a(ui-sref="me") Me App
+        li
+          a(ui-sref="current") Over ons
+        li 
+          a(ui-sref="contact") Contact
+      ul.nav.navbar-nav.navbar-right
+        li
+          a.login_btn(ng-click="openAuthPopup()")(ng-if="!$root.auth_user")
```

Comment:
/var/www/forus-frontend/src/sources/forus-platform/pug/tpl/directives/website/navbar.pug, line 31:
You should not have pug tags with multiple attributes.

)( should be removed